docs/features/schedulers.md

```diff
@@ -0,0 +1,45 @@
+# Schedulers
+
+Schedulers are instructor scripts that run periodically. They can be managed via `Manage Course > Manage schedulers`.
+
+!!! info "Interval Guarantees"
```

Comment:
Nit: Maybe make this a note?

Reply:
Resolved offline -- the convention in the docs are to use admonitions